packages/api/models/pdrs.js

```diff
@@ -5,6 +5,7 @@ const get = require('lodash/get');
 const log = require('@cumulus/common/log');
 const { getCollectionIdFromMessage } = require('@cumulus/message/Collections');
 const { getMessageExecutionArn } = require('@cumulus/message/Executions');
+const { getCreatedAt } = require('@cumulus/message/Message');
```

Comment:
This should be re-implemented like the feature-rds branch, this helper is already in message/src/workflows.ts.

Reply:
Update from feature branch: 55dd9454f